src/lib/features/playground/playground-service.ts

```diff
@@ -4,11 +4,20 @@ import { IUnleashServices } from 'lib/types/services';
 import { ALL } from '../../types/models/api-token';
 import { PlaygroundFeatureSchema } from 'lib/openapi/spec/playground-feature-schema';
 import { Logger } from '../../logger';
-import { IUnleashConfig } from 'lib/types';
+import { ISegment, IUnleashConfig } from 'lib/types';
 import { offlineUnleashClient } from './offline-unleash-client';
 import { FeatureInterface } from 'lib/features/playground/feature-evaluator/feature';
 import { FeatureStrategiesEvaluationResult } from 'lib/features/playground/feature-evaluator/client';
 import { ISegmentService } from 'lib/segments/segment-service-interface';
+import { FeatureConfigurationClient } from '../../types/stores/feature-strategies-store';
+import { generateObjectCombinations } from './generateObjectCombinations';
+
+type EvaluationInput = {
```

Comment:
introducing a type to capture what is actually used for the playground evaluation. Those are the values that we derive from the user input by going to our stores